unionml/model.py

```diff
@@ -514,15 +514,19 @@ def train(
 
         :param hyperparameters: a dictionary mapping hyperparameter names to values. This is passed into the
             ``init`` callable to initialize a model object.
-        :param loader_kwargs: key-word arguments to pass to the registered :meth:`unionml.Dataset.loader` function.
+        :param loader_kwargs: key-word arguments to pass to the registered 
```

Comment:
hi @bilal-aamer wanted to clarify the updates needed for https://github.com/unionai-oss/unionml/issues/148.

It needs to list the tasks that are invoked in the training workflow.

Can you revert the changes that you made below and add a paragraph below the argument docstrings, something like:

```
The train method invokes an execution graph that composes together the following functions to train and evaluate a model:

- :meth:`unionml.Dataset.reader`
- :meth:`unionml.Dataset.loader`
- :meth:`unionml.Dataset.splitter`
- :meth:`unionml.Dataset.parser`
- :meth:`unionml.Model.trainer`
- :meth:`unionml.Model.predictor`
- :meth:`unionml.Model.evaluator`
```

Reply:
Sure will do

Reply:
Done! @cosmicBboy ! Please Review!